airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -432,7 +432,7 @@ def download_data(self, url: str, chunk_size: int = 1024) -> tuple[str, str]:
         # set filepath for binary data from response
         tmp_file = os.path.realpath(os.path.basename(url))
         with closing(self._send_http_request("GET", f"{url}/results", stream=True)) as response, open(tmp_file, "wb") as data_file:
-            response_encoding = response.apparent_encoding or response.encoding or self.encoding
+            response_encoding = response.encoding or self.encoding
```

Comment:
can you clarify why you made this change?

Reply:
response.`apparent_encoding` method tries to guess encoding by reading the whole response content. this caused OOM for large responses.

Reply:
makes sense!